src/chi/components/input-wrapper.scss

```diff
@@ -22,6 +24,18 @@ $border: 0.0625rem;
     }
   }
 
+  &.-label-inline {
```

Comment:
@jllr @janibolkvadze naming convention is a little inconsistent. Thoughts on changing `-labeled` and `-label-inline` to `-labeled` and `-labeled--inline` to show a relationship between the two classes?

Reply:
This might require some more thought. If the purpose of `m-form__item` is to contain an input and a label and all `m-form__item` elements have a label, why do we need a `-labeled` and `-labeled--inline` class? Labeled toggle switches, labeled range sliders, checkboxes, and radio buttons render labels without the class. Only difference appears to be placement and font-weight.

Reply:
@janibolkvadze @mattnickles makes sense. It is not an easy approach. My thought is that as m-form__item usually contains a label and an input there should be a standard layout for it: column or row. For me the base would be the column one and the version would be the row. So we can create the standard component based in one behavior and with only one class change it.

Reply:
![Screenshot 2019-10-28 at 18 06 04](https://user-images.githubusercontent.com/25136131/67700290-ffeea880-f9ad-11e9-8268-9c18935de247.png)
This solves everything, but I think class name `-labeled` is misleading since m-form__items with inline positioned labels are also 'labeled'.

What if I make it something like:
```
  &.-labeled, &.-label-block {
    flex-direction: column;
  }
```
?